sdk/src/command-helper.js

```diff
@@ -77,37 +98,29 @@ class CommandHelper {
 
           ctx.streamed = command.streamed;
 
-          this.invokeHandler(() => handler(deserCommand, ctx), ctx, grpcMethod, reply => { return {reply}; }, "Command");
+          const reply = this.invokeHandlerLogic(() => handler(deserCommand, ctx), ctx, grpcMethod, "Command");
 
+          if (reply && reply.reply) {
```

Comment:
I haven't managed to get rid of those lines, any suggestion is welcome, anything I have tried was breaking the TCK.